src/bundle/Resources/public/scss/mixins/_drag-and-drop.scss

```diff
@@ -568,6 +568,17 @@
     }
 }
 
+@mixin drag-item-placeholder-static($border-color: $ibexa-color-info) {
```

Comment:
This is very unique situation that we avoid calculateRem(), I think we don't need to create a reusable mixin for it because we don't want such a thing to happen too often.